frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card.component.ts

```diff
@@ -0,0 +1,117 @@
+import type { OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { createSelector, Store } from '@ngrx/store';
+import type {
+    ExerciseRadiogram,
+    SimulatedRegion,
+} from 'digital-fuesim-manv-shared';
+import {
+    Client,
+    currentParticipantIdOf,
+    isAccepted,
+    isDone,
+    isUnread,
+    UUID,
+} from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import { map } from 'rxjs';
+import type { AppState } from 'src/app/state/app.state';
+import { selectOwnClientId } from 'src/app/state/application/selectors/application.selectors';
+import {
+    createSelectRadiogram,
+    selectClients,
+    selectSimulatedRegions,
+} from 'src/app/state/application/selectors/exercise.selectors';
+import { ExerciseService } from 'src/app/core/exercise.service';
+import { selectStateSnapshot } from 'src/app/state/get-state-snapshot';
+
+// Clients that leave are lost from the state but radiograms might point to them.
+// This is a fallback to show something useful in the UI
+const unavailableClient = Client.create('Unbekannt', 'participant');
+
+@Component({
+    selector: 'app-radiogram-card',
+    templateUrl: './radiogram-card.component.html',
+    styleUrls: ['./radiogram-card.component.scss'],
+})
+export class RadiogramCardComponent implements OnInit {
+    @Input() radiogramId!: UUID;
+    radiogram$!: Observable<ExerciseRadiogram>;
+    simulatedRegion$!: Observable<SimulatedRegion | undefined>;
+
+    ownClientId: UUID | undefined;
```

Comment:
Why can this be undefined? If it can't be, then don't check for it to be defined later

Reply:
I do not know when exactly this is set. The selector potentially returns undefined and I want to be better save that sorry with assumptions about that.

Reply:
It is only undefined when the client hasn't joined an exercise yet. See 'frontend/src/app/state/application/application.state.ts' So I think one can make the assumption that it is always set when that component is rendered

Reply:
Also, you could then remove the check whether it is defined from the Action sends

Reply:
Done